apps/api/src/app/message-template/usecases/update-message-template/update-message-template.usecase.ts

```diff
@@ -33,7 +33,7 @@ export class UpdateMessageTemplate {
       updatePayload.name = command.name;
     }
 
-    if (command.content) {
+    if (command.content !== null) {
```

Comment:
Allow the update of an empty string

Reply:
Thanks @djabarovgeorge, added the undefined in this [PR](https://github.com/novuhq/novu/pull/5302)